roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-/usr/bin/docker run --privileged --rm \
---net=host --pid=host --name=kubelet \
+/usr/bin/docker run --privileged \
+--net=host --pid=host --name=kubelet --restart=always \
```

Comment:
Not a bad thought. Updated that one and the ones that are present in the etcd templates.